pipelines/batch/setup_eval_job.py

```diff
@@ -108,7 +115,7 @@ def main(
                 "target": "/pyrenew-hew/params",
             },
             {
-                "source": "pyrenew-hew-prod-output",
+                "source": "pyrenew-test-output",
                 "target": "/pyrenew-hew/output",
```

Comment:
Noting the target of `output`

Reply:
Why are you noting the target of `output`? Are you uncertain about it?

Reply:
So I could link to it from the other comments.